internal/helm/repository/oci_chart_repository_test.go

```diff
@@ -125,44 +131,56 @@ func TestOCIChartRepoisitory_Get(t *testing.T) {
 			name:     "should return latest stable version",
 			version:  "",
 			expected: "1.0.0",
+			url:      "oci://localhost:5000/my_repo",
 		},
 		{
 			name:     "should return latest stable version (asterisk)",
 			version:  "*",
 			expected: "1.0.0",
+			url:      "oci://localhost:5000/my_repo",
 		},
 		{
 			name:     "should return latest stable version (semver range)",
 			version:  ">=0.1.5",
 			expected: "1.0.0",
+			url:      "oci://localhost:5000/my_repo",
 		},
 		{
 			name:     "should return 0.2.0 (semver range)",
 			version:  "0.2.x",
 			expected: "0.2.0",
+			url:      "oci://localhost:5000/my_repo",
 		},
 		{
 			name:     "should return a perfect match",
 			version:  "0.1.0",
 			expected: "0.1.0",
+			url:      "oci://localhost:5000/my_repo",
 		},
 		{
 			name:     "should return 0.10.0",
 			version:  "0.*",
 			expected: "0.10.0",
+			url:      "oci://localhost:5000/my_repo",
```

Comment:
It's the same URL string again and again. Let's use a constant/variable for it. The old `url` variable can just be moved up and reused in all these cases.

Reply:
Also, can you move `url` above `expected`? Just to have the expected fields at the bottom.